_episodes/01-design.md

```diff
@@ -17,41 +17,49 @@ see [the instructor training course][training].
 
 ## Reverse Instructional Design
 
+### Idealized
+
 In principle,
 we design lessons in four stages:
 
-1.  **Conceptual**:
-    describe who the lesson is for,
-    what its overall goals are,
+1.  **Conteptual:** describe target audience,
+    overall lesson's goals,
     and how long it is going to be.
-    For example,
-    the lesson might be for people who have taught themselves
+
+    **Example**:
+
+    a. A lesson for people who have taught themselves
     how to write page-long statistical analyses in R using RStudio,
     but have never written functions or run programs from the Unix shell prompt.
-    Its overall goal might be to teach them how to write modular multi-page programs
-    and how to use dplyr to regularize their analyses,
-    and the time allotted might be half a day.
+
+    b. Lesson's overall goal is to teach them how to write modular programs
+    and how to use `dplyr` to regularize their analyses.
+
+    c. Esimated time: half a day.
+
     It's often helpful to use [concept maps][concept-maps] in this stage.
 
-2.  **Summative Assessment**:
-    figure out what learners will do to demonstrate that they have mastered the material.
-    This is the most important step of the four,
-    since it is what *actually* determines the scope of the lesson.
-    In this case,
-    the summative assessment might be to write a four-function program
+2.  **Summative Assessment:**
+    figure out how learners will demonstrate that they have mastered the material.
+
+    _**This is the most important step**_ because
```

Comment:
Only boldface.

Reply:
why?

Reply:
```
$ cat strong.md 
_**This is the most important step**_ because
$ kramdown strong.md 
<p><em><strong>This is the most important step</strong></em> because</p>
$ cat italic.md 
**_This is the most important step_** because
$ kramdown italic.md 
<p><strong><em>This is the most important step</em></strong> because</p>
```

Semantically you should not use both because doesn't make sense. If is a minor emphasis you should use `<em>` and if is a strong emphasis you should use `<strong>`.

Reply:
OK